src/core/CMakeLists.txt

```diff
@@ -245,10 +245,10 @@ target_sources(openthread-radio PRIVATE
 
 target_link_libraries(openthread-ftd
     PRIVATE
-        mbedtls
+        openthread-mbedtls
```

Comment:
Make this public?

Reply:
I think it should not be public, because `mbedtls` headers are not exported by public headers of `openthread-ftd`.

Reply:
Maybe this should be public even though mbedtls headers is not exported by openthread-ftd, otherwise every target depends on openthread-ftd must know what it depends on, otherwise there will be link errors, which I think is not what cmake dependency system designs for. Thoughts?

Reply:
> otherwise every target depends on openthread-ftd must know what it depends on, otherwise there will be link errors

I don't think this is true for CMake. When we write
```cmake
target_link_libraries(foo PRIVATE bar)
```

I think CMake will link `bar` and all its dependencies for `foo`.

Reply:
yes, you are right. Thanks.